keras_vggface/tensorflow_lite_conversion.py

```diff
@@ -20,26 +22,45 @@ def create_tflite_model_file(keras_model, filename):
 
 
 def get_embeddings_from_png_image_example():
-    """Example usage"""
+    """Example usage to get face embeddings from cropped image of human face"""
     import numpy as np
     from tensorflow.keras.preprocessing import image
 
-    preprocessing_model = preprocessing(output_shape=(224, 224, 3))
-    model = VGGFace(model="senet50", pooling="avg", include_top=False, input_shape=(224, 224, 3))
+    image_preprocessor = create_vggface_preprocessing_model()
+    embeddings_model = VGGFace(model="senet50", pooling="avg", include_top=False, input_shape=(224, 224, 3))
 
     img = image.load_img('../image/ajb.jpg', target_size=(224, 224))
     x = image.img_to_array(img)
     x = np.expand_dims(x, axis=0)
-    preprocessed = preprocessing_model.predict(x)
-    embeddings = model.predict(preprocessed)
+    preprocessed = image_preprocessor.predict(x)
+    embeddings = embeddings_model.predict(preprocessed)
     print("embeddings: ", embeddings)
 
 
+def get_predictions_from_png_image_example():
+    """Example usage to get predictions (human identity) from image"""
+    from tensorflow.keras.preprocessing import image
```

Comment:
why are these imports inside a function?

Reply:
Oh i just read this :) 

Its because I was using this file in multiple places, and only need that import for an "example"/ test case, so I thought id reduce the dependencies (you don't need need image preprocessing if you just want to create the model).